### PR TITLE
Fix CI on older Elixir versions

### PR DESCRIPTION
**Pull request by whatyouhide**

### CODE REVIEW

File: priv/templates/message.ex.eex [hunk 1]
```diff
@@ -13,10 +13,12 @@ defmodule <%= @module %> do
   <% end %>
 
   <%= for v <- @oneofs do %>
-  <%= v %><% end %>
+  <%= v <> "\n" %>
```
Comment (ericmj) [inline priv/templates/message.ex.eex:16]: Nice solution!

File: test/protobuf/protoc/generator/enum_test.exs [hunk 2]
```diff
@@ -36,10 +36,15 @@ defmodule Protobuf.Protoc.Generator.EnumTest do
...
              field :A, 0
+
              field :B, 1
+
              field :HAS_UNDERSCORES, 2
+
              field :HAS_UNDERSCORES_X, 3
+
              field :HAS_UNDERSCORES_, 4
+
```
Comment (ericmj) [inline test/protobuf/protoc/generator/enum_test.exs:47]: We only get the double newlines in tests, right? The generated code won't have this?

**Review by ericmj — APPROVED** [review]

File: lib/protobuf/protoc/generator/util.ex [hunk 3]
```diff
@@ -3,6 +3,8 @@ defmodule Protobuf.Protoc.Generator.Util do
 
   alias Protobuf.Protoc.Context
 
+  @locals_without_parens [field: 2, field: 3, oneof: 2, rpc: 3, extend: 4, extensions: 1]
```
Comment (britto) [inline lib/protobuf/protoc/generator/util.ex:6]: We already have these settings exported in `.formatter.exs`. Is there a simple way to reuse them without hacking Mix too much?

**Review by britto — APPROVED** [review]
Neat! ✨

File: lib/protobuf/protoc/generator/util.ex (hunk 3 above)
Comment (whatyouhide) [reply lib/protobuf/protoc/generator/util.ex:6]: Nothing worth it yet IMO 🙃

**Pull request merged**